examples/simple_bucket/main.tf

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module "bucket" {
+  source = "../../modules/simple_bucket"
+
+  name     = "example-bucket"
+  project  = "example-project"
+  location = "us-east1"
+  iam_members = [{
```

Comment:
Instead of requiring users to set IAM roles themselves (and potentially misconfiguring IAM), could we use `bucket_admins`, `bucket_creators`, and `bucket_viewers` variables?

Reply:
Just to be clear, you mean helper iam fields in addition to iam_members?

If we require a helper field for every role it can get quite large and somewhat inflexible I imagine (esp if using custom roles, etc like in https://github.com/terraform-google-modules/terraform-google-cloud-storage/issues/18).

Also other modules like healthcare, etc also just have a single iam_members field.

Reply:
Fair point, I was mostly thinking about consistency with the existing module. I guess it's a nice-to-have but not required.

Agreed with should always support explicit `iam_members`.